src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -198,32 +199,36 @@ class EmojiPicker extends React.Component<IProps, IState> {
         [emoji.annotation, ...emoji.shortcodes, emoji.emoticon, ...emoji.unicode.split(ZERO_WIDTH_JOINER)]
             .some(x => x?.includes(filter));
 
-    private onEnterFilter = () => {
+    private onEnterFilter = (): void => {
         const btn = this.bodyRef.current.querySelector<HTMLButtonElement>(".mx_EmojiPicker_item");
         if (btn) {
             btn.click();
         }
     };
 
-    private onHoverEmoji = (emoji: IEmoji) => {
+    private onHoverEmoji = (emoji: IEmoji): void => {
         this.setState({
             previewEmoji: emoji,
         });
     };
 
-    private onHoverEmojiEnd = (emoji: IEmoji) => {
+    private onHoverEmojiEnd = (_emoji: IEmoji): void => {
```

Comment:
Tiny nitpick, why the change from `emoji` to `_emoji`

Reply:
So that it no longer complains about `unused variable emoji`. Found some prior art for it in https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/UserActivity.ts#L217.

Reply:
Ah, seems to make sense